### ISSUE TITLE
The file ./start-compose.sh no longer exists

### ISSUE THREAD

**Issue by sidahmed-malaoui** [n1nj4sec/pupy#833]
I used to launch pupy using the script `./start-compose.sh` (like mentioned in your documentation). But it looks like this file no longer exists.

Did you remove this functionality ? If so can you plz remove it from the documentation ! If not, can you tell me please how I can launch `pupy` now with docker-compose ?

PS : pupy rocks, really thank you for the author and contributors.

**Comment by alxchk** [comment]
Yes, recently this was removed. Right now there are no prebuilt templates, so you'll need to build it by your own. You can try to use create-workspace.py script to build templates and create docker/podman containers.

**Comment by AymC** [comment]
If it may help, I just put on pastebin the installation procedure that worked for me on last Saturday on a Kali 2019.4: https://pastebin.com/WMTSMLsu
Docker containers are run to build the templates, then I execute Pupy using <workdir>/bin/pupysh.

**Comment by isa84sfs** [comment]
@AymC 

Hi I followed the steps but I am still facing issues, would you be willing to help?
If yes then I will paste the details.

**Comment by AymC** [comment]
@isa84sfs 
Sure. Note that I followed successfully these steps on this Kali Linux version:
```
Linux kali 5.3.0-kali2-amd64 #1 SMP Debian 5.3.9-3kali1 (2019-11-20) x86_64 GNU/Linux
```
